lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -9,33 +9,37 @@ class DatabaseAdmin < HighLine
       NFS_FILE       = "Network File System (NFS)".freeze
       SMB_FILE       = "Samba (SMB)".freeze
       FILE_OPTIONS   = [LOCAL_FILE, NFS_FILE, SMB_FILE, CANCEL].freeze
-      FILE_MENU_ARGS = ["Restore Database File", FILE_OPTIONS, LOCAL_FILE, nil].freeze
 
       DB_RESTORE_FILE      = "/tmp/evm_db.backup".freeze
       LOCAL_FILE_VALIDATOR = ->(a) { File.exist?(a) }.freeze
 
-      NFS_PROMPT = <<-PROMPT.strip_heredoc.chomp
-        location of the remote backup file
-        Example: #{SAMPLE_URLS['nfs']}
-      PROMPT
-      SMB_PROMPT = <<-PROMPT.strip_heredoc.chomp
-        location of the remote backup file
-        Example: #{SAMPLE_URLS['smb']}
-      PROMPT
       USER_PROMPT = <<-PROMPT.strip_heredoc.chomp
         username with access to this file.
         Example: 'mydomain.com/user'
       PROMPT
 
-      attr_accessor :backup_type, :task, :task_params, :delete_agree, :uri
+      DB_DUMP_WARNING = <<-WARN.strip_heredoc
+        WARNING:  This is not the recommended and supported way of running a
+        database backup, and is strictly meant for exporting a database for
+        support/debugging purposes!
+
+        Please use the 'Backup' functionality instead.
```

Comment:
I think this last line should be qualified in some way or just removed. Out of context it feels like you should never be using this option.

Reply:
Yeah, this was meant to be a draft, and I figured there would have been some thoughts around the actual copy of it.

I did want to point them to the #Correct™ way of doing things, but didn't want to fall into my trap of getting to wordy and write another 📖 .  I am fine with deleting it as well, though I guess I would prefer to keep it for the aforementioned reasoning, but don't really have a good qualifier in mind.  My only though was:

```
Please use the 'Backup' functionality instead if you intend
to create a backup to restore in application failure and
disaster recovery scenarios.
```